src/app/camo-card/camo-card.component.ts

```diff
@@ -1,11 +1,15 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, Input, OnInit } from '@angular/core';
+import { Progress } from '../home-page/Progress';
+import { Weapon } from 'data/data-definitions'
```

Comment:
I added the weapon import from data-definitions.ts part cuz of the input but idk if it's correct or not